modules/userId/eids.js

```diff
@@ -375,6 +375,16 @@ export const USER_IDS_CONFIG = {
         };
       }
     }
+  },
+  'rubicon': {
+    source: 'rubiconproject.com',
+    atype: 1,
+    getUidExt: function(data) {
+      return data.ext;
+    },
+    getValue: function getValue(data) {
```

Comment:
I think we can just leave this line at `getValue: function(data) {` for consistency with the other USER_IDS_CONFIG entries.